core/migrations/3_deploy_timer.js

```diff
@@ -0,0 +1,8 @@
+const Timer = artifacts.require("Timer");
+const { getKeysForNetwork, deploy } = require("../../common/MigrationUtils.js");
+
+module.exports = async function(deployer, network, accounts) {
+  const keys = getKeysForNetwork(network, accounts);
+
+  await deploy(deployer, network, Timer, { from: keys.deployer });
```

Comment:
nit: shouldn't we only deploy this contract if `controllableTiming` is `true`?

Reply:
aha yep